### PR TITLE
Add support for nginxinc

### PR DESCRIPTION
**Pull request by bbucher-sigsci**
This PR will add support for using the [nginxinc ingress controller](https://github.com/nginxinc/kubernetes-ingress).

I'm following the same process that already exists for supporting [kubernetes nginx ingress controller](https://github.com/kubernetes/ingress-nginx).

TODO:
* need to add additional information to the README, and customer docs (they're both the same.)

I'd like to merge these changes in order to see the workflow run in Github. I've already tested the workflow locally with [nektos](https://github.com/nektos/act), which passed.

### CODE REVIEW

File: .github/workflows/release-and-pr-nginxinc-ingress.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    - uses: actions/checkout@v2
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_API_TOKEN }}
+    - name: compare versions
+      id: tag
+      env:
+        REPO: signalsciences/sigsci-nginxinc-nginx-ingress-controller
```
Comment (leaf-fastly) [inline .github/workflows/release-and-pr-nginxinc-ingress.yml:30]: could `sigsci-nginxinc-nginx-ingress-controller`  be `sigsci-nginxinc-ingress-controller`?

File: .github/workflows/release-and-pr-nginxinc-ingress.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        echo PR_NUM=$pr_number >> $GITHUB_ENV
+
+    - name: Add reviewers to latest PR
+      if: ${{ steps.tag.outputs.NO_CHANGE == 'false' }}
+      run: |
+        curl -s \
+        -H "Accept: application/vnd.github.v3+json" \
+        -u ${{ github.actor }}:${{ secrets.GITHUB_TOKEN }} \
+        ${{ env.GH_REPO }}/pulls/${{ env.PR_NUM }}/requested_reviewers \
+        --data '{"reviewers":["bbucher-sigsci","leaf-fastly","rell-fastly","andy-fastly"]}'
```
Comment (leaf-fastly) [inline .github/workflows/release-and-pr-nginxinc-ingress.yml:89]: probably don't want `andy-fastly` here

File: README.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ Tags/Releases track image tags of the upstream docker repo.
 These images work as drop-in replacements for:
 * `k8s.gcr.io/ingress-nginx/controller` images with corresponding tags (0.43.0)
 * `quay.io/kubernetes-ingress-controller/nginx-ingress-controller` images with corresponding tags (0.25.0-0.33.0)
+* `nginx/nginx-ingress` images with corresponding tags (>= 2.2.0)
```
Comment (leaf-fastly) [inline README.md:10]: I think you need to call this out separately, as it's coming from a different dockerhub repo.

File: sigsci-nginxinc-values.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+controller:
+
+  image:
+    ## Replaces the default nginx-controller image with a custom image that contains the Signal Sciences nginx Module
+    repository: signalsciences/sigsci-nginxinc-nginx-ingress-controller
```
Comment (leaf-fastly) [inline sigsci-nginxinc-values.yaml:5]: (reminder to update this if you remove the second `nginx` from above)

**Review by leaf-fastly — CHANGES_REQUESTED** [review]
Really the doc change (to call out the separate dockerhub repo) and removing andy-fastly from reviewers are the things that need to change; the repo name `nginx` redundancy is just my opinion.

File: README.md (hunk 3 above)
Comment (bbucher-sigsci) [reply README.md:10]: Separated that line from the rest, and added additional instructions specific to using nginxinc/kubernetes-ingress controller

File: sigsci-nginxinc-values.yaml (hunk 4 above)
Comment (bbucher-sigsci) [reply sigsci-nginxinc-values.yaml:5]: done

File: .github/workflows/release-and-pr-nginxinc-ingress.yml (hunk 2 above)
Comment (bbucher-sigsci) [reply .github/workflows/release-and-pr-nginxinc-ingress.yml:89]: removed

File: .github/workflows/release-and-pr-nginxinc-ingress.yml (hunk 1 above)
Comment (bbucher-sigsci) [reply .github/workflows/release-and-pr-nginxinc-ingress.yml:30]: done and changed name in dockerhub

**Review by leaf-fastly — APPROVED** [review]

**Pull request merged**